lib/features/genai_setting/bloc/genai_bloc.dart

```diff
@@ -7,6 +7,7 @@ import 'package:public_chat/service_locator/service_locator.dart';
 import 'package:public_chat/utils/bloc_extensions.dart';
```

Comment:
changes in this file is not related to this issue

Reply:
When I tested the app I didn't see any loading state while waiting for Gemini response so I added this feature. I will remove it now and create a new PR when needed